pnpm-lock.yaml

```diff
@@ -702,13 +702,13 @@ importers:
         version: 7.23.1
       '@commercetools-test-data/commons':
         specifier: 6.4.1
-        version: link:../commons
+        version: 6.4.1
       '@commercetools-test-data/core':
         specifier: 6.4.1
-        version: link:../../core
+        version: 6.4.1
       '@commercetools-test-data/utils':
         specifier: 6.4.1
-        version: link:../../utils
```

Comment:
I don't know where these changes are coming from, I used `pnpm@8.10.2` as stated in `package.json`